airbyte-integrations/connectors/source-zendesk-sell/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 982eaa4c-bba1-4cce-a971-06a41f700b8c
-  dockerImageTag: 0.1.0
+  dockerImageTag: 0.1.1
```

Comment:
@speckm can you bump the connector doc changelog entry? After we can merge the contribution!

Reply:
I added a changelog entry für version 0.1.1.